sql/scanner/token.go

```diff
@@ -115,14 +115,27 @@ const (
 	WHERE
 	WRITE
 
-	TYPEARRAY
-	TYPEBLOB
-	TYPEBOOL
-	TYPEBYTES
-	TYPEDOCUMENT
-	TYPEDOUBLE
-	TYPEINTEGER
-	TYPETEXT
+	ARRAY
+	BLOB
+	BOOL
+	BYTES
+	DOCUMENT
+	DOUBLE
+	INTLITERAL // literal INTEGER
```

Comment:
Sorry, now I remember why there was this `TYPE` prefix, it is to distinguish a type keyword and values.
We should use actually the `TYPE`  prefix for all the types

Reply:
Concerning the `PRECISION` keyword, it should be in the same group as the other normal keywords above, ordered alphabetically.